modules/ROOT/pages/tokenization.adoc

```diff
@@ -24,6 +27,16 @@ A connection is established between the API gateway and the tokenization service
 
 The API gateway replaces the credit card number with the tokenized data and the request is sent to the app. 
 
+== Masking
+
+Anypoint Security also provides masking which is another tokenization option that can be used to hide or mask identity. When using masking, a configurable mask character is returned which hides the identity of the sensitive data. 
+
+In the above example, using a masking format that preserves the last four characters option would return the token: `\####\##########--4567`. The identity of a masked token cannot be returned via a detokenization call to the tokenization service.
+
+[WARNING]
+====
+If you configure a policy in API Manager to detokenize and you select a masking format, the API gateway returns a `400 Bad Request` response to the API. The API gateway returns this response because the masking format only masks data based on the original value. It cannot unmask data and return original value.
```

Comment:
```suggestion
If you configure a policy in API Manager to detokenize and you select a masking format, the API gateway returns a `400 Bad Request` response to the API. The API gateway returns this response because the masking format only masks data based on the original value. It cannot unmask data and return the original value.
```

Reply:
Fixed